uint8 robot_watcher
uint8 nodes_status